README.md

```diff
@@ -13,11 +13,25 @@ Every raw message contains important parts:
 
 session alias, direction and sequence number are a compound unique identifier of raw messages within th2
 
+# Configuration
+
+```json
+{
+  "drain-interval": 1000,
+  "termination-timeout": 5000
+}
+```
+
+#### drain-interval
+Interval in milliseconds to drain all aggregated batches that are not stored yet. The default value is 1000.
+
+#### termination-timeout
+The timeout in milliseconds to await the inner drain scheduler has finished all tasks. The default value is 5000.
```

Comment:
The timeout in milliseconds to await **for** the inner drain scheduler **to** finished all **the tasks**. The default value is 5000.